uat/custom-components/src/main/java/com/aws/greengrass/utils/UnitConvertHelper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.utils;
+
+import ch.qos.logback.core.util.FileSize;
+
+public final class UnitConvertHelper {
+    public UnitConvertHelper() {
+    }
+
+    public static int getFileSizeInBytes(double fileSize, String fileSizeUnit) {
```

Comment:
Just put all this methods on the same class.

Reply:
This will be used by another component